keras_core/utils/python_utils.py

```diff
@@ -118,7 +118,7 @@ def remove_long_seq(maxlen, seq, label):
     """
     new_seq, new_label = [], []
     for x, y in zip(seq, label):
-        if len(x) < maxlen:
+        if len(x) <= maxlen:
```

Comment:
We should keep `<` here. This function only exists for backwards compat with tf.keras so it should follow the same semantics.